### ISSUE TITLE
Double Feature Request (One QoL)

### ISSUE THREAD

**Issue by torma616** [esjeon/krohnkite#91]
Hi! Double feature request incoming:

_QoL Request:_ Quarter Tiling automatically creates new desktop when 4 windows are tiled 
_Addendum: This could possibly be used for other layouts too, if you include a "Max Windows per Layout" option for users to set._

_Other request:_ Any possibility of adding a BSP style tiling layout?

If no to both of these, thanks anyway! Love the script!

**Comment by esjeon** [comment]
The first one has been a part of my goal. It's just that it has been a lower priority + I have less free time these days.

In case of layouts, I'll definitely add more, but only after refactoring, which I'm working on these days.

Btw, what is BSP layout?

**Comment by torma616** [comment]
BSP = Binary Space Partitioning, although I don't mean specifically that. What I'm thinking of is also sometimes called Spiral or Fibonacci.

Here are some examples if I suck at explaining what I'm thinking about: 
https://cdn.niu.moe/media_attachments/files/001/664/147/original/58dc79c5c082395f.png
http://thedarnedestthing.com/images/bspwm_populated.jpg
https://i.imgur.com/D4WhUaz.png
https://i.ytimg.com/vi/9ofq4gpG_lM/maxresdefault.jpg

**Comment by 10zero11** [comment]
I would also like to +1 request for Binary Space Partitioning (BSP).

Another example is how this implemented in yabai for MacOs.
  https://github.com/koekeishiya/yabai

many thanks for Krohnkite. :)

**Comment by TamasBarta** [comment]
bspwm/yabai are good examples. I also would love to see that feature.
